cesium/internal/controller/controller.go

```diff
@@ -436,5 +468,5 @@ func (c *Controller[E]) remove(r *region[E]) {
 }
 
 func Unauthorized(name string, ch core.ChannelKey) error {
-	return errors.Wrapf(control.Unauthorized, "writer %s does not have control authority over channel %s", name, ch)
+	return errors.Wrapf(control.Unauthorized, "writer %s does not have control authority over channel %d", name, ch)
```

Comment:
Don't know if you do this at the unary level, but would be good to add channel name here as well